lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -231,34 +231,34 @@ switchport_autostate_exclude:
   default_value: false
 
 switchport_mode_dot1q_tunnel:
+  _exclude: [cli_ios_xr]
   default_only: ""
 
 switchport_mode_ethernet:
-  cli_nexus:
-    auto_default: false
-    config_get_token_append: '/^(?:no )?switchport mode ?(.*)$/'
-    config_set_append: "<state> switchport mode <mode>"
-    default_value: "access"
-  cli_ios_xr:
-    default_only: ""
+  _exclude: [cli_ios_xr]
```

Comment:
Will the return nil work if you don't do exclude but explicitly just do nexus, for example?